### ISSUE TITLE
Is it possible to zoom in zoom out like google maps?

### ISSUE THREAD

**Issue by bdsamson** [dabeng/OrgChart#175]
where it zooms on the mouse cursor

**Comment by TechnicalRhino** [comment]
you can use "zoom" option in the orgchart option you provide.

> `$('#some-chart-container').orgchart({
> 		'nodeContent' : 'title',
> 		'pan' : true,
> 		'zoom' : true
> });`

**Comment by bdsamson** [comment]
Yeah I know that it is zoomable, but is it possible to zoom in where the mouse cursor is? the zoom functionality is centered in the middle.
